src/installation.md

```diff
@@ -0,0 +1,180 @@
+   2. [Development Environment](#most-advanced-version-development-environment-version)
+
+We will guide you how to install for each software to be ready to use our engine. Before you go into the installaion, please open the links that are provided above.
+
+## Rust
+When you visit the page for Rust, it will greet you the screen as below.
+![Rust Install Page](/images/rust_install_page.png)
+
+Please follow the instructions as they stated. The page might look slighlty different based on operating system.
+
+## CMake
+When you visit the [page for Cmake](https://cmake.org/download/), it will greet you the screen as below. Please download the latest version of Cmake that has the option of Windows x64 Installer: the file extension is `.msi`.
+![Cmake Install Page](/images/cmake_install_page.png)
+In the image above, the version we are downloading is 3.26.4, but you can download and install a higher version if you want as well.
+
+Once you download the installer, open the installer. The first screen should be similar as the following screen.
+
+![Cmake Installer Part 1](/images/cmake_installer_part_1.png)
+
+Select `Next` to continue the installation process.
+
+The next screen will ask regarding the agreement of the program. Select agree to continue the installation process as the image below. You can save the agreement by selecting the `print` option if you want.
+
+![Cmake Installer Part 2](/images/cmake_installer_part_2.png)
+
+In the next screen, we are have an option to install on system path for current user, all users, or not at all. Choose either for the current user or all users to add the system PATH. The image below has selected for all users.
+
+![Cmake Installer Part 3](/images/cmake_installer_part_3.png)
+
+The next screen is selection the program path. The image below has selected the default path, but you can choose whatever path you want.
+
+![Cmake Installer Part 4](/images/cmake_installer_part_4.png)
+
+The next screen is an verifier on selecting all the correct options that you have selected. Once everything is all set, click the `install` button to proceed the installation.
+
+![Cmake Installer Part 5](/images/cmake_installer_part_5.png)
+
+Once the installation is complete, then you will see the following screen.
+
+![Cmake Installer Part 6](/images/cmake_installer_part_6.png)
+
+## Oculus VR
+Once you [open the page](https://www.oculus.com/Setup/), you will be greet the follow page as below.
+
+![Oculus Download Page](/images/oculus_download_page.png)
+
+Select the `Download Oculus Rift Software` button. By selecting the button will open an `.exe`. Please follow the installer instruction.
+
+## Steam VR
+//TODO: someone test this out/write this part. I have no idea what is a major difference when it comes to Steam VR application.
+[Download Link](https://store.steampowered.com/app/250820/SteamVR/)
+
+## Our Engine
+There are two methods to use our engine: the stable version and most advanced version.
+
+### Stable Version
```

Comment:
I think the "Stable Version" should appear first on this page, since that's what most users will want. Maybe move the rest of the compilation instructions to it's own section (or maybe it's own page?)